__tests__/config.test.ts

```diff
@@ -50,15 +50,19 @@ describe('Config', () => {
 
   it('throw error when config does not exist', async () => {
     const tempPath = generateTempPath();
-    function getConfig() {
-      readConfig('development', tempPath);
+    async function getConfig() {
+      await readConfig('development', tempPath);
+    }
+    try {
+      await getConfig();
+    } catch (e) {
+      expect(e).toEqual(
```

Comment:
we need to use .rejects to make sure its throwing error - https://jestjs.io/docs/expect#rejects